akka-testkit-typed/src/main/scala/akka/testkit/typed/javadsl/TestProbe.scala

```diff
@@ -112,17 +92,15 @@ abstract class TestProbe[M] {
    * }
    * }}}
    */
-  def within[T](min: FiniteDuration, max: FiniteDuration)(f: Supplier[T]): T =
+  def within[T](min: FiniteDuration, max: FiniteDuration, f: Supplier[T]): T =
```

Comment:
let's use java.time.Duration in javadsl instead, since we are changing to that anyway in other places